django_olcc/olcc/management/commands/olccimport.py

```diff
@@ -71,10 +71,19 @@ def product_from_row(self, row):
                 if row.get('proof'):
                     product.proof = row.get('proof')
                 if row.get('age'):
-                    if row.get('age_unit') == 'M':
-                        product.age = int(float(row.get('age'))) / 12
+                    if 'age_unit' in row:
```

Comment:
Can we just drop the old age unit entirely?

Reply:
I was wondering the same. 🙂 `age_unit` is still a valid header within [PriceHistory_CSV.csv](http://www.olcc.state.or.us/pdfs/PriceHistory_CSV.csv). However, it looks like that CSV hasn't been updated since March of 2015. It seems like there's no point in preserving the ability to parse it.

Do you think it would be alright to remove `import_csv_price_history` and `import_price_history`?

Reply:
Hrm, I'm not sure. Probably? I think they've changed file formats a few times over the years and I can't remember what those were used for.

Reply:
It seems to me like it would only be useful for testing, assuming OLCC never updates it again. It could be a good way to populate with historic data since it's got a couple years worth. But even then, it seems like you'd be better off just using a fixture. I'll update my PR to remove it.

Somewhat related, is something running `olccfetch` on a timer? I couldn't find anything in the code. That's part of what made me think twice about removing price history (in case something I can't see was still using it).

Reply:
> It could be a good way to populate with historic data since it's got a couple years worth.

Yes. I think that's what I was using it for previously.

> Somewhat related, is something running `olccfetch` on a timer?

Yep! There's a Heroku scheduler running that task daily.

<img width="776" alt="screen shot 2018-11-05 at 9 12 31 am" src="https://user-images.githubusercontent.com/107117/48014210-06b84780-e0db-11e8-83af-35cfc3b003a6.png">